### PR TITLE
[workspace] fix the UI of delete modal in the save objects(assets) page

### PR DESCRIPTION
**Pull request by Qxisylolo**
### Description
This pr fixes the UI of delete modal in the save objects(assets) page by [issue 300-303]
1. Change "saved objects" to "assets" in the Delete modal.
2. Swap the "ID" column with the title column and make the modal wider
3. fix the bug of pagination alignment and remove the bullet point
4. Capitalize both letters in "ID" column title

## Screenshot

before:
截屏2024-09-19 15 20 39

after:

## Changelog
- fix: fix the UI of delete modal in the save objects(assets) page

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by SuZhou-Joe** [comment]
@kgcreative So no matter new home feature is turned on or off, we will use the term `assets` instead of `saved objects`?

**Comment by kgcreative** [comment]
> @kgcreative So no matter new home feature is turned on or off, we will use the term `assets` instead of `saved objects`?

Yes -- we may need to alias some of the old references though

File: src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx [hunk 1]
```diff
@@ -864,80 +864,84 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
...
-                ),
-              },
-            ]}
-            pagination={true}
-            sorting={false}
-          />
-        </EuiConfirmModal>
+            </EuiButtonEmpty>
+
+            <EuiButton type="submit" onClick={onConfirm} fill color="danger">
```
Comment (AMoo-Miki) [inline src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx:930]: Should we disable the button while `Deleting...`?

**Review by AMoo-Miki — DISMISSED** [review]

File: src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx (hunk 1 above)
Comment (ruanyl) [reply src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx:956]: Makes sense to me. cc @Qxisylolo

Comment (Qxisylolo) [reply src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx:956]: Thanks for all your comments, already updated.

File: src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx [hunk 2]
```diff
@@ -890,80 +890,90 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
...
-          />
-        </EuiConfirmModal>
+            </EuiButtonEmpty>
+
+            <EuiButton
+              type="submit"
+              onClick={onConfirm}
+              fill
+              color="danger"
+              disabled={isDeleting ? true : false}
```
Comment (SuZhou-Joe) [inline src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx:961]:
```suggestion
              disabled={!!isDeleting}
```

**Review by SuZhou-Joe — DISMISSED** [review]

**Review by ruanyl — DISMISSED** [review]

**Review by Hailong-am — APPROVED** [review]

**Review by ruanyl — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Pull request merged**